src/lang/extra/english.txt

```diff
@@ -1406,11 +1400,30 @@ STR_FINANCES_REPAY_TOOLTIP_EXTRA                                :{BLACK}{STRING}
 STR_FINANCES_BORROW_QUERY_CAPT                                  :{WHITE}Enter the amount of money to borrow
 STR_FINANCES_REPAY_QUERY_CAPT                                   :{WHITE}Enter the amount of money to repay
 
-STR_INDUSTRY_VIEW_TRANSPORTED_TOOLTIP                           :{BLACK}{STRING}{RAW_STRING}
-STR_INDUSTRY_VIEW_TRANSPORTED_TOOLTIP_EXTENSION                 :{}{BLACK}{CARGO_LONG} ({COMMA}%)
+STR_NEW_LINE                                                    :{}
 
 # Town tooltip
-STR_TOWN_NAME_TOOLTIP                                           :{BLACK}{TOWN}
+STR_TOWN_NAME_TOOLTIP                                           :{BLACK}{TOWN}{RAW_STRING}
+STR_TOWN_NAME_POP_TOOLTIP                                       :{BLACK}{TOWN}: {COMMA}{RAW_STRING}
+STR_TOWN_NAME_RATING_TOOLTIP                                    :{}Your rating is {STRING}
+
+STR_INDUSTRY_VIEW_NAME_TOOLTIP                                  :{STRING}
+STR_INDUSTRY_VIEW_REQUIRED_TOOLTIP_FIRST                        :{STRING}{RAW_STRING}
+STR_INDUSTRY_VIEW_REQUIRED_TOOLTIP_NEXT                         :, {STRING}{RAW_STRING}
+STR_INDUSTRY_VIEW_STOCKPILED_TOOLTIP                            :{CARGO_LONG} waiting{RAW_STRING}
+STR_INDUSTRY_VIEW_TRANSPORTED_TOOLTIP_EXTENSION                 :{CARGO_LONG}{RAW_STRING} ({COMMA}%)
+STR_INDUSTRY_VIEW_INFO_TOOLTIP                                  :{BLACK}{RAW_STRING}
+
+STR_DEPOT_VIEW_COUNT_TOOLTIP                                    :{COMMA} indside
+STR_DEPOT_VIEW_TOTAL_TOOLTIP                                    :Of {COMMA} inside:
```

Comment:
These are misspelled. Also would it be worth expanding these a bit ({COMMA} vehicles inside)?